packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -109,12 +110,19 @@ export class UiControlClient extends FluentCommand {
     }
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  private escape_separator_string(instruction: string): string {
+    return instruction.split(Separators.STRING).join('"');
+  }
+
   async exec(
     instruction: string,
     customElementJson: CustomElementJson[] = [],
   ): Promise<void> {
-    const customElements = await CustomElement.fromJsonListWithImagePathOrImage(customElementJson);
     const { secretText } = this;
+    const customElements = await CustomElement.fromJsonListWithImagePathOrImage(customElementJson);
+    const stringWithoutSeparators = this.escape_separator_string(instruction);
```

Comment:
You have written too much Python 😆 
```suggestion
    const stringWithoutSeparators = this.escapeSeparatorString(instruction);
```

Reply:
das stimmit :laughing: